crates/texlab/src/util.rs

```diff
@@ -11,13 +11,19 @@ pub fn normalize_uri(uri: &mut lsp_types::Url) {
         if let Some(mut path) = segments.next().and_then(fix_drive_letter) {
             for segment in segments {
                 path.push('/');
-                path.push_str(segment);
+                path.push_str(&segment);
```

Comment:
This seems unnecessary :)

The `segment` is already a `&str`, so further referencing is not needed.

Reply:
Yeah, definitely :)